tap_azure_git/__init__.py

```diff
@@ -268,7 +294,10 @@ def load_schemas():
             schema = json.load(file)
             refs = schema.pop("definitions", {})
             if refs:
-                singer.resolve_schema_references(schema, refs)
+                schema = singer.resolve_schema_references(
```

Comment:
This `resolve_schema_references` wasn't doing anything since we didn't actually set the schema to the result.

In `jsonschema` the `$ref` items are resolved by complete paths inside the json document.  Singer does not handle them in that way, but this transformation allows us to validate the json schema documents with `ajv-cli` and also support `--discover` .